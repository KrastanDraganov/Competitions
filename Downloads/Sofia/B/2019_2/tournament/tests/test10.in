100
57 1 3 4 5 6 9 10 11 12 13 16 18 19 23 24 26 28 31 32 33 34 37 43 45 47 48 49 50 52 54 55 56 57 61 62 64 65 68 69 70 72 73 76 77 82 84 86 87 88 89 90 92 95 96 97 98 99
53 3 4 5 6 9 10 12 13 14 16 19 23 26 31 33 34 37 38 39 41 43 44 45 46 47 49 50 52 54 55 56 57 60 61 64 65 66 68 69 70 72 73 76 77 82 84 86 87 88 95 96 97 99
81 0 1 3 4 5 6 7 9 10 11 12 14 15 16 17 18 19 20 21 23 24 25 26 27 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 47 48 50 51 52 53 54 55 56 57 59 61 62 63 65 66 67 68 69 70 71 72 73 74 75 76 77 78 80 82 84 86 87 90 91 92 93 94 95 97 99
22 9 12 13 19 32 33 43 50 55 61 72 74 75 76 82 84 87 88 90 91 96 99
48 3 5 6 9 10 12 13 16 19 20 21 23 26 28 31 32 34 37 41 43 44 47 49 50 52 54 55 56 57 61 64 65 66 70 72 73 76 77 84 87 88 90 93 94 95 96 97 99
44 3 6 9 10 12 13 15 19 23 26 27 28 31 32 33 34 41 43 44 45 47 49 51 52 55 59 60 61 64 66 68 72 73 76 77 82 84 87 88 90 93 96 97 99
37 3 9 12 13 14 19 26 28 31 32 33 34 43 44 45 46 47 49 50 52 55 61 62 64 72 73 75 76 77 82 84 88 90 92 94 97 99
76 0 1 3 4 5 6 9 10 12 13 14 15 16 17 18 19 22 23 24 25 27 28 29 30 31 32 33 34 36 38 39 40 41 43 44 45 47 48 49 50 52 53 54 55 56 57 59 61 62 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 80 82 84 86 90 91 92 95 96 97 98 99
89 0 1 2 3 4 5 6 7 9 10 12 13 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 77 78 79 80 83 84 85 86 87 88 89 91 92 93 94 95 96 97 98 99
13 19 24 26 35 39 43 55 59 61 64 69 77 99
31 3 6 9 12 13 19 21 26 31 32 33 34 43 47 49 50 54 55 62 67 72 73 77 80 82 84 88 90 96 97 99
86 1 3 4 5 6 7 8 9 10 12 13 14 15 17 18 19 20 21 22 23 24 25 27 28 29 30 31 32 34 35 36 37 38 39 40 41 43 44 45 46 47 48 49 50 51 53 54 55 56 57 58 60 61 62 63 64 65 66 69 70 71 72 73 74 75 76 77 78 80 81 82 83 84 85 86 87 88 89 91 92 93 94 95 96 98 99
21 9 19 28 32 33 34 43 44 50 55 72 76 77 82 84 86 87 88 90 97 99
29 2 9 12 17 19 31 32 33 38 43 44 45 47 51 52 55 61 64 72 73 74 79 82 84 87 88 90 97 99
49 0 3 4 5 8 9 10 12 13 16 19 23 28 31 32 33 34 37 40 41 43 44 45 47 49 52 54 55 56 57 62 64 65 68 69 70 72 73 76 77 79 82 84 87 88 90 95 96 99
60 0 1 3 4 6 8 9 10 12 13 14 16 19 22 23 26 28 30 31 33 34 37 39 41 43 44 45 47 49 50 51 52 53 54 55 56 57 58 61 62 63 64 65 66 68 69 70 72 76 82 84 86 87 88 90 93 95 96 97 99
49 3 5 6 9 10 11 12 13 19 22 23 28 31 32 33 34 37 41 43 44 45 47 48 49 52 54 55 57 61 64 65 66 68 70 72 73 74 76 77 82 84 88 90 91 94 95 96 97 99
66 0 1 3 4 5 6 9 10 12 14 15 16 18 19 23 24 25 26 27 28 30 31 32 33 34 36 37 38 39 40 43 44 45 47 48 49 50 52 54 55 56 57 62 63 64 65 66 67 68 69 70 72 73 75 76 77 84 86 87 90 93 95 96 97 98 99
59 1 3 4 5 6 9 10 12 13 14 15 16 19 23 24 26 28 30 31 33 34 37 39 41 43 44 45 47 48 49 50 52 54 55 56 57 61 62 64 65 66 68 69 70 72 73 76 77 82 86 87 88 90 92 95 96 97 98 99
12 37 43 44 53 55 61 67 72 77 82 96 99
77 0 1 3 5 6 7 9 10 12 13 14 15 16 17 18 19 21 22 23 24 25 26 27 28 31 32 33 36 37 38 39 40 41 42 44 45 47 48 49 50 52 53 54 55 57 59 61 62 63 64 65 66 67 69 70 71 73 75 76 77 78 80 82 84 86 87 88 90 91 92 93 94 95 96 97 98 99
74 0 1 3 5 6 7 9 12 13 14 15 16 17 18 19 23 24 27 28 30 31 32 33 34 35 36 38 39 40 41 42 44 45 47 48 49 50 52 53 54 55 56 57 59 61 62 63 65 66 68 69 70 71 72 73 74 75 77 78 80 82 84 86 87 88 90 91 92 93 94 95 96 98 99
81 0 1 2 3 4 5 6 9 10 12 13 14 17 18 19 21 23 24 25 26 27 28 29 30 31 32 33 34 36 37 38 39 40 41 42 43 44 45 48 49 50 52 53 54 55 56 57 58 59 61 62 63 65 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 84 86 88 90 91 92 93 94 95 96 97 98 99
36 3 6 9 10 12 13 19 26 30 31 32 33 34 39 44 45 47 49 50 52 54 61 63 70 72 73 76 77 82 84 87 88 90 96 97 99
52 1 3 4 5 6 10 12 13 14 15 16 19 23 26 28 32 33 34 37 39 43 44 45 47 49 50 52 54 55 56 57 61 62 65 66 68 69 70 72 73 76 77 81 82 86 87 88 89 96 97 98 99
67 0 1 3 4 5 6 9 10 12 13 14 15 16 18 19 21 23 24 26 28 30 31 32 33 36 37 38 39 40 43 44 45 47 49 50 51 53 54 55 56 57 61 62 63 64 66 67 68 69 70 73 74 75 76 77 82 84 86 87 88 90 92 95 96 97 98 99
32 3 7 11 12 13 14 16 19 21 27 31 32 33 36 43 44 45 50 52 55 61 63 72 73 76 77 82 84 87 90 97 99
66 0 1 3 4 6 9 10 12 13 14 15 16 18 19 23 24 25 28 30 32 33 34 36 37 38 39 40 41 43 44 45 49 50 52 53 54 55 56 59 61 62 63 64 65 66 68 69 70 72 74 75 76 77 82 84 86 87 88 90 91 92 95 96 97 98 99
37 1 2 3 9 10 13 19 23 26 30 31 32 33 34 43 44 45 47 49 50 52 55 61 64 72 73 76 77 82 84 87 88 90 91 96 97 99
80 0 1 2 3 4 5 6 9 10 12 13 14 15 16 17 18 19 20 21 23 24 25 26 27 28 30 31 32 34 37 38 39 40 41 43 44 45 47 48 50 52 53 54 55 56 57 58 59 60 61 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 82 84 86 87 88 90 91 92 93 94 98 99
58 0 1 3 4 5 6 9 10 12 13 14 16 19 20 24 26 31 32 33 34 36 38 39 41 44 45 47 48 50 52 53 54 55 56 57 61 62 64 65 66 68 69 70 73 74 76 77 82 84 86 87 88 90 95 96 97 98 99
29 3 8 9 12 19 24 27 32 33 39 42 43 44 45 50 55 57 61 64 72 76 77 82 84 87 88 90 97 99
17 1 8 9 15 18 19 33 38 43 46 47 55 61 72 77 82 99
22 4 9 11 19 29 43 44 55 61 71 76 77 80 82 84 86 87 88 90 91 97 99
34 3 9 13 19 20 25 26 31 32 33 43 44 45 46 47 49 50 52 55 61 64 66 72 73 77 80 82 84 85 87 88 90 97 99
87 0 1 3 4 5 6 7 10 12 13 14 15 16 17 18 19 20 22 23 24 25 26 27 28 29 30 31 32 33 34 36 37 38 40 43 44 45 46 47 48 49 50 51 52 53 54 55 56 58 59 60 61 62 63 64 65 66 67 68 70 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 98 99
62 0 1 3 4 5 6 9 10 12 13 14 15 16 18 19 23 24 28 29 31 32 33 34 37 39 41 43 44 48 50 52 53 55 56 57 58 60 61 62 63 64 65 66 68 69 70 72 73 76 77 80 82 84 86 88 90 92 95 96 97 98 99
41 3 5 6 7 9 10 12 13 21 23 26 28 30 31 32 33 34 39 43 45 47 49 50 52 54 56 57 61 66 68 69 72 73 76 77 82 84 87 88 97 99
61 0 3 4 5 6 9 10 12 14 15 16 18 19 23 24 26 28 31 33 34 36 37 39 40 41 43 44 45 47 48 49 50 52 53 54 55 56 57 61 62 63 64 66 68 69 70 72 73 75 77 82 86 87 88 90 92 95 96 97 98 99
50 0 3 4 5 6 10 12 13 14 16 19 26 28 32 33 34 35 41 43 44 45 46 49 50 52 54 55 57 61 62 64 66 68 69 70 71 72 73 76 77 79 86 87 88 90 92 95 96 97 99
67 0 1 3 4 5 6 9 10 12 13 15 16 18 19 23 24 26 28 30 31 32 33 34 36 37 39 41 43 44 45 47 48 49 50 52 53 54 55 56 57 61 62 64 65 66 68 69 70 71 72 73 76 77 80 82 84 85 86 87 88 90 92 95 96 97 98 99
46 0 3 6 9 10 12 13 17 19 23 24 25 26 28 31 32 33 34 35 37 43 44 45 47 49 50 52 55 61 63 64 65 66 72 73 76 77 81 82 84 87 88 90 96 97 99
82 0 1 3 4 5 6 7 9 10 11 12 13 14 15 16 17 18 19 23 24 25 26 27 28 29 30 32 33 34 35 36 37 38 39 40 41 43 44 45 47 48 49 50 52 53 54 55 56 57 59 61 62 63 64 65 66 67 68 69 70 71 72 73 75 77 78 80 82 84 86 87 88 90 91 92 93 94 95 96 97 98 99
16 20 21 23 30 52 55 59 72 73 77 82 85 86 88 97 99
19 0 3 9 10 32 37 43 50 55 56 61 70 72 76 77 82 84 91 99
27 3 4 9 10 12 19 32 33 36 43 44 46 50 51 55 58 61 72 76 77 82 83 84 87 88 90 99
81 0 2 3 4 5 7 9 10 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 33 36 37 38 40 41 42 43 44 47 48 49 50 51 52 53 54 55 56 57 59 61 62 63 64 65 67 68 69 70 71 73 74 75 76 77 78 79 80 82 86 87 88 89 90 92 93 94 95 96 97 98 99
33 3 9 12 19 22 26 27 31 33 36 39 43 44 45 50 52 54 55 61 64 67 72 73 76 77 82 84 85 87 88 90 97 99
62 1 3 4 5 6 9 10 12 13 14 15 19 23 24 25 26 27 28 31 32 33 34 37 39 41 43 44 45 47 49 50 52 54 55 56 57 58 61 62 64 65 66 68 69 70 72 73 76 77 79 82 83 84 86 87 88 90 95 96 97 98 99
33 2 3 9 12 13 19 26 29 30 31 32 33 36 43 44 45 47 50 52 55 60 61 70 72 73 77 82 84 87 88 90 97 99
21 5 9 13 14 16 19 32 33 43 61 72 73 76 77 82 84 87 88 90 97 99
81 0 1 3 4 6 7 9 10 12 14 16 17 18 19 20 21 22 23 24 26 27 28 29 30 31 32 33 34 36 37 38 39 40 41 42 43 44 47 48 49 50 52 53 54 55 56 57 58 59 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 84 86 87 88 90 91 92 93 94 95 97 98 99
25 3 9 10 11 12 19 25 31 32 33 44 45 50 58 61 72 76 77 82 84 87 88 90 97 99
61 0 1 3 4 5 6 8 9 10 12 13 14 16 17 18 23 24 26 28 31 32 33 34 37 39 41 43 44 45 47 48 49 50 52 54 55 56 57 62 64 65 66 68 69 70 72 73 76 77 78 82 84 86 87 88 90 92 95 96 98 99
45 3 5 6 9 12 13 19 26 28 31 32 33 34 36 41 43 44 45 49 50 52 55 56 57 59 60 61 64 65 66 69 72 73 76 77 82 83 86 87 88 90 93 96 97 99
11 23 37 50 52 61 68 71 72 73 87 99
44 3 5 6 9 10 12 13 16 19 20 23 26 28 31 32 33 34 39 41 43 45 47 49 50 52 55 61 64 66 67 68 71 72 73 76 77 82 84 88 90 94 95 97 99
42 3 5 6 9 10 12 13 19 23 26 27 28 32 33 34 35 41 43 44 45 47 49 50 52 55 56 61 64 66 68 72 73 76 77 82 84 87 88 90 96 97 99
80 0 1 2 3 4 5 6 7 9 10 12 13 14 16 17 18 19 20 21 23 24 25 26 27 28 30 31 32 33 34 37 38 39 40 41 42 43 44 46 47 49 50 53 54 55 56 57 59 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 77 78 80 82 84 86 87 88 90 91 92 93 95 96 97 98 99
70 0 1 3 4 6 10 11 12 13 14 15 16 17 18 19 23 24 25 26 28 30 31 32 33 34 36 37 38 39 40 41 44 45 47 48 49 50 52 53 55 56 57 61 62 63 64 65 66 67 68 69 72 73 74 75 76 77 78 82 84 86 87 88 90 91 95 96 97 98 99
85 0 2 3 4 6 7 9 10 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 30 31 32 33 34 37 38 39 40 41 42 43 44 45 46 47 48 50 51 52 53 55 56 57 58 59 62 63 64 65 66 67 68 69 70 73 74 75 76 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 97 98 99
12 10 12 14 17 43 53 60 69 71 73 78 81
49 1 3 4 5 9 12 13 16 19 23 26 28 29 31 32 33 34 37 41 43 44 45 47 49 50 52 54 55 56 57 61 64 65 66 68 69 70 72 73 76 77 82 84 87 90 95 96 97 99
59 0 1 3 4 5 6 7 9 10 12 13 14 16 18 19 24 28 30 31 32 33 34 37 39 40 43 44 45 47 48 49 50 52 53 54 55 56 57 61 62 65 66 67 68 69 70 72 73 76 82 84 86 88 90 95 96 97 98 99
36 2 3 10 12 19 21 22 23 24 26 32 33 37 43 44 45 49 50 52 55 61 63 72 74 76 77 78 82 84 87 88 90 91 94 97 99
45 3 5 6 9 10 12 13 19 23 25 26 28 31 32 33 34 37 38 39 43 44 45 47 49 50 52 55 56 57 61 64 66 68 72 73 77 79 82 84 87 88 90 93 96 97
36 0 3 6 9 10 12 13 14 19 22 23 26 28 31 32 33 43 44 45 46 47 49 50 52 55 61 64 72 77 82 84 87 88 90 96 99
64 0 1 3 4 5 6 9 11 12 13 14 15 16 18 21 22 23 24 26 27 28 30 31 32 33 34 36 37 38 39 40 41 43 44 45 48 49 50 52 53 54 55 57 61 62 64 65 66 68 69 72 73 74 75 76 77 82 84 86 90 92 96 97 98
42 3 4 6 9 10 11 12 13 19 20 23 26 28 31 32 33 34 41 43 44 45 47 49 50 52 54 61 64 66 72 73 76 77 81 82 84 87 88 90 96 97 99
47 3 4 5 6 10 12 13 16 19 23 26 28 31 32 33 34 35 41 43 44 45 47 49 50 52 55 56 57 64 65 66 68 70 71 72 73 76 77 82 84 87 88 90 95 96 97 99
45 3 5 6 9 10 12 13 19 26 28 31 32 33 34 37 41 43 45 47 50 52 54 55 56 57 59 61 64 65 66 67 68 72 73 76 77 82 83 84 87 88 90 95 96 99
69 0 1 3 4 5 6 9 10 12 13 14 15 16 17 18 19 23 24 25 26 27 28 30 31 32 34 35 36 37 38 41 43 44 45 47 48 49 50 52 53 54 57 59 60 62 63 64 65 66 67 68 70 72 74 75 76 77 78 82 84 86 87 88 90 91 92 95 96 98
10 9 20 25 30 33 46 60 61 78 85
29 3 9 12 15 19 27 31 32 33 44 45 52 64 66 71 72 76 77 82 84 85 86 87 88 90 93 94 97 99
67 0 1 4 5 6 9 10 12 14 15 17 18 19 20 23 24 26 28 31 32 33 34 36 37 38 39 40 41 42 43 44 45 47 48 49 50 52 53 54 55 56 57 61 62 63 65 66 68 69 70 72 73 75 76 77 82 84 86 87 88 90 92 95 96 97 98 99
65 0 1 4 5 9 10 12 13 14 15 16 18 19 23 24 26 28 30 31 32 33 34 36 37 39 40 41 43 44 45 47 48 49 50 52 53 54 55 56 57 61 62 63 64 65 66 68 69 70 72 73 76 77 82 84 86 87 88 90 92 95 96 97 98 99
24 8 9 10 13 19 21 32 34 38 42 43 49 55 58 61 65 66 72 77 82 83 84 96 99
12 3 13 15 55 60 61 63 72 78 82 90 99
70 0 1 3 4 5 6 9 10 12 13 14 15 16 17 18 19 23 24 25 26 27 28 30 31 32 33 34 36 37 38 39 40 41 43 44 45 47 48 49 50 52 54 55 56 57 62 63 65 66 67 68 69 70 73 74 75 76 82 84 85 86 87 88 90 92 95 96 97 98 99
83 0 1 2 3 4 5 6 7 9 10 11 12 15 16 17 18 19 20 21 23 24 25 26 27 28 30 31 32 33 34 36 37 38 40 41 42 43 44 45 47 49 50 52 53 54 55 56 57 58 59 61 62 63 64 66 67 68 69 70 71 72 73 74 75 76 77 78 80 82 84 86 87 88 90 91 92 93 94 95 96 97 98 99
74 0 1 3 4 5 6 9 12 13 14 15 16 17 18 19 23 24 25 26 27 28 30 31 32 37 38 39 41 43 44 45 47 48 49 50 51 52 53 54 55 56 57 59 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 82 83 84 86 87 90 91 92 95 96 97 98 99
89 0 1 2 3 4 5 6 7 8 9 10 12 13 14 15 16 17 18 19 20 21 23 25 26 27 28 29 30 31 32 33 34 36 37 38 39 40 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 62 63 64 65 66 67 69 70 71 72 73 74 75 76 77 78 79 80 82 83 84 86 87 89 90 91 92 93 94 95 96 97 98 99
10 4 8 9 17 39 51 55 61 72 98
84 0 1 2 3 4 5 6 7 9 10 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 36 37 38 39 40 41 42 43 44 46 47 49 50 51 52 53 55 56 57 58 59 61 62 63 64 65 66 67 68 69 71 72 73 74 75 77 78 79 82 84 86 87 88 90 91 92 93 96 97 98 99
18 9 18 19 24 32 38 39 43 46 54 55 61 72 77 82 93 98 99
86 0 1 2 3 4 5 6 7 9 10 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 36 37 38 39 41 42 44 45 46 48 49 50 51 52 53 54 55 56 57 58 59 61 62 63 64 65 66 67 68 69 70 71 74 75 76 77 79 80 81 82 83 84 86 87 88 89 90 91 92 93 96 97 98 99
46 3 4 5 6 9 10 13 14 16 19 23 26 28 31 32 34 37 41 44 45 47 49 50 52 55 56 57 61 62 64 65 66 68 69 70 72 76 77 82 84 88 90 95 96 97 99
24 6 7 9 10 16 19 22 32 36 43 44 56 61 63 67 72 76 77 82 84 86 91 93 99
24 2 7 9 17 19 26 32 44 55 61 62 67 72 76 77 80 81 82 84 87 91 95 97 99
90 1 2 3 4 5 6 7 9 10 12 13 14 15 16 17 18 19 20 21 22 23 25 26 27 28 29 30 31 32 33 34 36 37 38 39 40 41 42 43 44 45 47 48 49 50 51 52 53 54 55 56 57 58 59 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 82 83 84 86 87 88 90 91 92 93 94 95 96 97 98 99
21 1 8 9 11 19 24 32 37 43 44 55 61 72 76 82 84 87 88 91 97 99
65 0 1 4 5 6 9 10 12 13 14 15 17 18 19 23 24 25 26 30 31 32 34 36 37 38 39 40 41 43 45 46 47 48 49 50 52 53 54 55 56 57 61 62 63 65 66 67 68 69 70 72 73 74 75 76 77 78 82 84 86 92 95 96 98 99
62 1 3 4 5 9 10 12 13 14 15 16 17 19 23 24 26 28 30 31 32 33 34 37 41 43 44 45 47 48 49 50 52 54 55 56 57 59 61 62 63 64 65 66 68 69 70 72 73 76 77 82 84 86 87 88 90 94 95 96 97 98 99
71 0 1 3 6 7 9 10 12 13 14 16 18 19 23 24 25 26 27 28 30 31 32 33 34 36 37 38 39 40 41 43 44 45 47 48 49 50 52 53 55 56 57 59 61 62 63 64 66 67 68 69 70 71 72 74 75 76 77 78 80 82 86 88 90 91 92 94 95 96 97 98
76 0 1 3 5 7 9 10 12 13 14 15 17 18 19 23 24 25 26 27 28 30 31 32 33 34 36 37 38 39 40 41 43 44 45 47 48 49 50 52 53 54 55 57 58 59 61 62 63 65 66 67 68 69 70 71 72 74 75 76 77 78 80 82 83 84 85 86 87 88 90 91 95 96 97 98 99
48 3 5 6 9 10 12 13 19 23 24 26 28 29 31 32 33 34 37 41 43 44 45 47 49 50 52 54 55 57 61 64 65 66 67 68 72 73 76 77 82 83 84 85 87 90 96 97 99
36 2 6 9 12 13 26 29 31 32 33 34 37 43 44 45 47 49 50 51 52 55 56 60 61 64 72 73 77 82 84 87 88 90 97 98 99
26 3 9 11 14 19 21 29 32 35 44 45 53 55 61 66 70 71 72 76 77 82 84 87 91 98 99
53 1 2 3 4 5 6 9 10 12 13 14 15 16 19 23 26 28 31 32 33 34 37 39 41 43 44 45 47 49 50 52 54 55 56 57 61 62 64 65 66 68 69 70 72 73 76 77 86 87 88 90 95 99
7 61 65 67 71 72 82 93
