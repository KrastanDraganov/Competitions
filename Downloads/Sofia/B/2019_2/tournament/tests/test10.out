94 95 85 89 93 92 96 90 88 81 83 79 80 97 91 74 75 73 71 87 86 84 77 60 67 68 76 66 46 51 42 65 64 63 62 57 35 38 40 53 48 54 56 49 47 45 36 34 55 50 33 29 30 20 22 31 27 25 28 26 21 17 44 32 18 24 16 11 15 14 8 23 13 2 7 0 1 4 5 10 6 3 12 9 19 37 39 41 43 52 58 59 61 69 70 72 78 82 98 99 
